src/phoenix/experimental/evals/functions/classify.py

```diff
@@ -381,13 +383,20 @@ def run_evals(
         as model invocation parameters and details about retries and snapping to
         rails.
 
-        concurrency (int, optional): The number of concurrent evals if async
-        submission is possible.
+        concurrency (Optional[int], default=None): The number of concurrent evals if async
+        submission is possible. If not set, the default is determined on a per-model basis.
 
     Returns:
         List[DataFrame]: A list of dataframes, one for each evaluator, all of
         which have the same number of rows as the input dataframe.
     """
+    # use the minimum default concurrency of all the models
+    if concurrency is None:
+        if len(evaluators) == 0:
+            concurrency = 1
+        else:
+            concurrency = min(evaluator.default_concurrency for evaluator in evaluators)
```

Comment:
```suggestion
    if not evaluators:
        return []

    concurrency = concurrency or min(evaluator.default_concurrency for evaluator in evaluators)
```

Reply:
This makes me wonder if we really should allow the user to use multiple different models inside `run_evals`.

Reply:
I think it's fine so far? I think this is a pretty small concession that we can probably work around